internal/pkg/template/input/validation.go

```diff
@@ -92,41 +98,42 @@ func validateInputIf(fl goValidator.FieldLevel) bool {
 }
 
 // Some input Kinds require specific Type of the input.
-func validateUserInputTypeByKind(value interface{}, kind string) error {
+func validateUserInputTypeByKind(value interface{}, kind string, fieldName string) error {
 	inputType := reflect.TypeOf(value).String()
 	switch kind {
 	case KindPassword, KindTextarea:
 		if inputType != reflect.String.String() {
-			return fmt.Errorf("the input is of %s kind and should be a string, got %s instead", kind, inputType)
+			return fmt.Errorf("%s is of %s kind and should be a string, got %s instead", fieldName, kind, inputType)
```

Comment:
Ostatní validace jsou upravený aby obsahovaly název fieldu.

Reply:
super, nazov fieldu tam potrebujeme.

Nebolo by lepsie tam ten `kind` nedavat, to je nejaka nasa interna vec, aky input zobrazime...
`input is of confirm kind and should be a bool, got int instead`
`input should be a bool, got int`

Ak uzivatel zadava hodnoty cez nas interaktivny CLI form, tak tam sa nemoze stat, ze to nebude `bool`.
Ak bude zadavat hodnotu cez `API/flag/nejaky JSON subor` ... tak tam ho bude zaujimat iba to, ze to ma byt bool,
to ze to vykreslujeme ako nejaky `confirm` uzivatela nezaujima. Podobne aj pri ostatnych hlaskach.

Reply:
jo 🙂 máš pravdu 👍